src/open_samus_returns_rando/pickups/pickup.py

```diff
@@ -151,8 +154,8 @@ def patch_model(self, model_names: list[str], bmsad: dict) -> None:
                 else:
                     MODELUPDATER["functions"][0]["params"].pop("Param2")
                 # Placeholder until custom models/textures are made
-                # Checks for caption in multiworld if for another world
-                if item_id in RESERVE_TANK_ITEMS or "Reserve Tank" in self.pickup["caption"]:
+                # FIXME: Make this less ugly
+                if model_name in RESERVE_TANKS or item_id in RESERVE_TANKS:
```

Comment:
I thought about asking to remove the `item_id` part but then I realised that's for compatibility with old patcher files, right?

Reply:
Yes but idk how important that is. If we ended up making a completely new model, then it'd be wrong anyway.

Reply:
Actually this isn't even going to work at all, because first it's checking if it's a tank model, then checking for the new model names. So the first check fails. I hate this part of the pickup code lol